nftables_test.go

```diff
@@ -2487,6 +2487,104 @@ func TestGetRuleLookupVerdictImmediate(t *testing.T) {
 	}
 }
 
+func TestDynset(t *testing.T) {
+	// Create a new network namespace to test these operations,
+	// and tear down the namespace at test completion.
+	c, newNS := openSystemNFTConn(t)
+	defer cleanupSystemNFTConn(t, newNS)
+	// Clear all rules at the beginning + end of the test.
+	c.FlushRuleset()
+	defer c.FlushRuleset()
+
+	filter := c.AddTable(&nftables.Table{
+		Family: nftables.TableFamilyIPv4,
+		Name:   "filter",
+	})
+	forward := c.AddChain(&nftables.Chain{
+		Name:     "forward",
+		Table:    filter,
+		Type:     nftables.ChainTypeFilter,
+		Hooknum:  nftables.ChainHookForward,
+		Priority: nftables.ChainPriorityFilter,
+	})
+
+	set := &nftables.Set{
+		Table:      filter,
+		Name:       "dynamic-set",
+		KeyType:    nftables.TypeIPAddr,
+		HasTimeout: true,
+		Timeout:    600 * 1000, // Timeout is specified in milliseconds
```

Comment:
Can you add this comment also to the Set type’s Timeout field documentation please? Would it make sense to use a time.Duration and document that we pass it to the kernel with millisecond-precision?

Reply:
Good idea! I should have thought of that.. Was focusing on functionality, rather than ease of use. I  will switch to use time.Duration.

Reply:
Addressed, please check it out in the latest push.